Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor

```diff
@@ -36,22 +44,31 @@
                        data-bs-target="@(!ParentSidebar.Collapsed ? "#" + id : null)"
                        aria-expanded="@(this.ShouldBeExpanded ? "true" : "false")">
                     <div class="hx-sidebar-item-navlink-content">
-                        @if (Icon != null)
+                        @if (ContentTemplate is null)
+                        {
+                            if (Icon != null)
+                            {
+                                <HxIcon Icon="Icon" />
+                            }
+                            <span class="text">
+                                @Text
+                            </span>
+
+                            <HxIcon Icon="@BootstrapIcon.ChevronRight" CssClass="expand-icon" />
+                        }
+                        else
                         {
-                            <HxIcon Icon="Icon" />
+                            @ContentTemplate
                         }
-                        <span class="text">
-                            @Text
-                        </span>
-                        <HxIcon Icon="@BootstrapIcon.ChevronRight" CssClass="expand-icon" />
```

Comment:
We should not drop the chevron-icon for version with `@ContentTemplate`, see above.